services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -182,3 +188,19 @@ sealed trait IntWorkflowQueryKey extends WorkflowQueryKey[Option[Int]] {
   def displayName: String
 }
 
+sealed trait BooleanWorkflowQueryKey[A] extends WorkflowQueryKey[Boolean] {
+  override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[Boolean] = {
+    valuesFromMap(grouped) match {
+      case vs if vs.lengthCompare(1) > 0 => s"Found ${vs.size} values for key '$name' but at most one is allowed.".invalidNel[Boolean]
+      case Nil => true.validNel
```

Comment:
ToL: Maybe have `def defaultValue` be a trait method on `BooleanWorkflowQueryKey` to allow the default to change depending on query key?